lib/main/atom/tooltips/tooltipView.tsx

```diff
@@ -42,21 +43,20 @@ export class TooltipView implements JSX.ElementClass {
   }
 
   public render() {
+    const code = this.props.info ? (
+      <div class="atom-typescript-tooltip-tooltip-code">
+        {this.props.info.displayString.replace(/^\(.+?\)\s+/, "")}
```

Comment:
Yeah I added it because now, the info in that parenthesized is provided in the `kind` section of the tooltip (due to factorization, the two tooltips are now identical). I decided to cut it out because it looked redundant to me.

Reply:
Ah, okay, that makes sense.